frontend/Components/Header/adminlink/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import styles from './styles';
+
+const AdminLink = () => {
+  return (
+    <div className={styles.adminPageContainer}>
+      <Link to="http://localhost:8080/admin">Admin</Link>
```

Comment:
I don't think we want to have the `http://localhost:8080` as part of the URL. You want whatever url you use here to work across different environments (say if we deployed to a test or production environment). Generally you only want the url path for that reason.

It should work if you just do `/admin`

Reply:
changed!